recipes/lsdeluxe/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "lsdeluxe" %}
+{% set version = "0.21.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Peltoche/lsd/archive/{{ version }}.tar.gz
+  sha256: f500c18221f9c3fd45f88f6f764001e99cf9d6d74af9172cbb9a9ff32f3e5c7d
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  ignore_run_exports_from:
    - {{ compiler('cxx') }}
```
Overdepending according to link inspector. Probably binary doesn't use CXX standard library.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=497376&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1150